src/boot_O2_g3/fault.c

```diff
@@ -751,8 +750,10 @@ void Fault_ProcessClients(void) {
 }
 
 #ifdef NON_MATCHING
-// regalloc and ordering differences around the two bool variables (faultCustomOptions and faultCopyToLog)
+// needs in-function static bss
 void Fault_SetOptionsFromController3(void) {
+    static u32 faultCustomOptions;
+
     Input* input3;
```

Comment:
We typically don't separate `static` and local variables
```suggestion
    static u32 faultCustomOptions;
    Input* input3;
```